### PR TITLE
update to 7.3.318

### PR DESCRIPTION
**Pull request by cori**
Also change install command to account to toggl's change to an exe installer rather than an msi and some updates to attribution in the nuspec file

### CODE REVIEW

**Comment by cori** [comment]
It seems to me that this package would be ripe for automation. I'd be happy to create a pull request to make that change, but I only have a loose understanding of how your automation works. Is that documented anywhere that I could review to get it set up and working locally so I could test those changes?

Also I'd love to see this package be able to take a packageParameter for which channel to use (stable vs beta) but I don't want to put the effort in unless it's something you'd be interested in merging, since based on my research it's not a trivial change.

Thanks!

**Comment by ferventcoder** [comment]
> Also I'd love to see this package be able to take a packageParameter for which channel to use (stable vs beta) 

This is not a package parameter concern. This is a package version concern. Am I right in assuming that beta has a beta version attached to it, such as 1.0.0-beta1? That means their is a package for each version you can install, including packages for beta versions. 

Does that make sense? Am I off base here on what you mean?

File: manual/toggl/toggl.nuspec [hunk 1]
```diff
@@ -3,13 +3,12 @@
   <metadata>
     <id>toggl</id>
     <title>Toggl Desktop</title>
-    <version>7.1.145</version>
-    <authors>Apprise LLC</authors>
+    <version>7.3.318</version>
+    <authors>Toggl OÜ</authors>
```
Comment (ferventcoder) [inline manual/toggl/toggl.nuspec]: Was this intentional?

File: manual/toggl/tools/chocolateyInstall.ps1 [hunk 2]
```diff
@@ -1 +1 @@
-﻿Install-ChocolateyPackage 'toggl' 'msi' '/quiet' 'https://toggl.com/api/v8/installer?app=td&platform=windows&channel=stable'
+﻿Install-ChocolateyPackage 'toggl' 'exe' '/quiet' 'https://toggl.com/api/v8/installer?app=td&platform=windows&channel=stable'
```
Comment (ferventcoder) [inline manual/toggl/tools/chocolateyInstall.ps1]: Do the silent args need adjusted here?

Comment (cori) [inline manual/toggl/tools/chocolateyInstall.ps1]: I don't believe so in my original testing, but let me pack and test this change to verify; I'll post back when done or update.

File: manual/toggl/toggl.nuspec (hunk 1 above)
Comment (cori) [inline manual/toggl/toggl.nuspec]: Yes. Apprise LLC makes the iOS app, but the desktop app is maintained by Toggl.

Comment (ferventcoder) [inline manual/toggl/toggl.nuspec]: I meant Toggl **0Ü**

**Comment by cori** [comment]
As far as the beta version is concerned it installs from 'https://toggl.com/api/v8/installer?app=td&platform=windows&channel=beta' and just installs a more recent app version than what's published in the stable download channel. See http://toggl.github.io/toggldesktop; I'm on the beta channel and just have v 7.3.335. 

That said, there certainly could be a beta package; it just seems like overkill but if that's the preferred way then no problem.

File: manual/toggl/toggl.nuspec (hunk 1 above)
Comment (cori) [inline manual/toggl/toggl.nuspec]: Ah. Well that's their legal name per their TOS (https://www.toggl.com/legal/terms see the Supplier definition). They're based in Estonia.

Comment (ferventcoder) [inline manual/toggl/toggl.nuspec]: Perfect. TIL

**Comment by ferventcoder** [comment]
> That said, there certainly could be a beta package; it just seems like overkill but if that's the preferred way then no problem.

My concern is that the package version needs to match the install. If that stable version of 7.3.335 matches the beta binaries for 7.3.335 (e.g. they don't change it at all), then we have some options here. However it sounds like they probably don't release an actual of 7.3.335 (unless it is good to go, then they promote it up to stable) for all versions in the beta channel. 

Both of these options still jive really well with a beta package pointing to beta channel, so users can choose to opt into betas in choco in a consistent manner with other packages.

**Comment by cori** [comment]
Ok that clarifies things and makes sense. I'll make up a beta Toggl package.

**Comment by ferventcoder** [comment]
For now, is /quiet good?

**Comment by cori** [comment]
nope - i was wrong about that - fixed now - sorry for the delay.

One last question on the beta issue - is that approach also preferred for insider channels or the like?

**Comment by ferventcoder** [comment]
Insider channels?

Like Windows insider channels?

**Comment by cori** [comment]
Yeah like the inside channel for visual studio code, for instance.

**Comment by ferventcoder** [comment]
What would you see for choco in this regard?

**Comment by cori** [comment]
In retrospect this is probably really a non-question - insider builds really aren't that different than betas, so following that same process is probably the most suitable approach.

I would still like to learn more about your automation framework - but i'll take that to gitter - don't want to _further_ hijack this PR for unrelated items :-)

**Comment by ferventcoder** [comment]
Pushed to the community repository, awaiting moderation.

**Pull request merged**